app/sites/all/modules/custom/layout/layout.module

```diff
@@ -102,10 +107,18 @@ function layout_block_view($delta = '') {
       $block['subject'] = '';
       $block['content'] = variable_get_lang('appset_frontpage_top', '');
       break;
+
     case 'main_menu':
       $block['subject'] = '';
       $block['content'] = layout_block_main_menu();
       break;
+
+    case 'promo_block':
+      if (user_is_anonymous()) {
```

Comment:
czemu `if`? Chyba można zrobić to samo w edycji bloku i wybrać kto ma dostęp. Chyba bardziej drupalowo :D